AmplifyPlugins/Auth/AWSCognitoAuthPlugin/Service/AWSMobileClient/AWSMobileClientAdapter.swift

```diff
@@ -100,6 +100,15 @@ class AWSMobileClientAdapter: AWSMobileClientBehavior {
                                    completionHandler)
     }
 
+    @available(iOS 13, *)
+    func showSignIn(uiwindow: UIWindow,
```

Comment:
Here and below:

Don't we already have a typealias like "AWSMobileClientPresentationContext" or similar for UIWindow? Aren't we requiring presentation context in other calls? If so, seems like we should make that consistent.